puppet/modules/quickstack/manifests/pacemaker/cinder.pp

```diff
@@ -9,6 +9,8 @@
   $backend_glusterfs_name = 'glusterfs',
   $backend_iscsi          = false,
   $backend_iscsi_name     = 'iscsi',
+  $backend_netapp         = false,
```

Comment:
This comment is _on pacemaker/cinder.pp.  You take the params, but then you dont do anything with them. Looking again, perhaps quickstack::cinder doesnt need anything extra, but you need to at least take the netapp params passed in here, and push them down into the call to cinder_volume within this file.  That is located here:
https://github.com/rhefner1/astapor/blob/enable-cinder-netapp/puppet/modules/quickstack/manifests/pacemaker/cinder.pp#L220